core/field.ts

```diff
@@ -1101,7 +1113,7 @@ export abstract class Field implements IASTNodeLocationSvg,
    *
    * @returns Element to bind click handler to.
    */
-  protected getClickTarget_(): Element {
+  protected getClickTarget_(): Element|null {
```

Comment:
this seems like one where if it's null, something has gone wrong and you could conceivably put the error here instead of at every calling location.

Reply:
I actually did so initially, though tests broke when I did so.